docs/index.md

```diff
@@ -1,6 +1,10 @@
 HTCondor-CE
 ===========
 
+!!! info "HTCondor-CE 3 Documentation"
+    This documentation is for HTCondor-CE 3.
+    If you need documentation for HTCondor-CE 4, please consult <https://htcondor.readthedocs.io/en/latest/> instead.
```

Comment:
I think you might have the wrong link here... this is the Condor manual.

Reply:
Good catch, fixed